mkdocs.yml

```diff
@@ -1,34 +1,38 @@
 site_name: CodeSee Beta
 nav:
-  - 👋 Introduction: index.md
-  - 🛠 Installing CodeSee:
-    - Installation and setup: installation.md
-    - Create React App: setup-cra.md
-    - Typescript without Babel: setup-typescript-without-babel.md
-    - Next: setup-next.md
-    - Nuxt: setup-nuxt.md
-    - Storybook: setup-storybook.md
-    - Gatsby: setup-gatsby.md
-    - Ember: setup-ember.md
-    - Generic setup: setup-generic.md
-    - Configuration options: options.md
+  - 👋 Welcome:
+    - Introduction: index.md
+    - 🙏 Troubleshooting: troubleshooting.md
```

Comment:
Shall we remove this emoji to keep the subpages' not having them consistent?

Reply:
Sounds good. Removed.